connectivity/access_jobs.sql

```diff
@@ -3,40 +3,38 @@
 -- location: cambridge
 ----------------------------------------
 -- low stress access
-UPDATE  cambridge_census_blocks
+UPDATE  neighborhood_census_blocks
 SET     emp_low_stress = (
             SELECT  SUM(blocks2.jobs)
             FROM    cambridge_census_block_jobs blocks2
```

Comment:
No it does not. There is another task to 'generalize scripts' which would involve renaming all of the tables other than `neighborhood_boundary` and `neighborhood_census_blocks` to remove the cambridge prefix or change it. I did not do that here so that the scope of changes remained reasonable.